config.md

```diff
@@ -46,24 +46,24 @@ The runtime MUST mount entries in the listed order.
 *Example*
 
 ```json
-"mounts": [
+  "mounts": [
     {
-        "name": "proc",
-        "path": "/proc"
+      "name": "proc",
+      "path": "/proc"
```

Comment:
Most of our other JSON examples use four-space indents, so we probably just want a blanket `s/^/    /` on this block instead of dropping to two-space indents.